tests/v1/FitnessDemo3/run_test

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env php
+<?php
+declare(strict_types=1);
+error_reporting(E_ALL);
+
+$PWD = __DIR__;
+
+// Must load autoloader before including any Annotation Processors
+if (file_exists($autoloaderFilePath = dirname($PWD, 5) . '/autoload.php')) {
+    /** @noinspection PhpIncludeInspection */
+    require_once $autoloaderFilePath;
+} else if (file_exists($autoloaderFilePath = dirname($PWD, 3) . '/vendor/autoload.php')) {
+    /** @noinspection PhpIncludeInspection */
+    require_once $autoloaderFilePath;
+} else {
+    throw new RuntimeException('Unable to find the Composer autoloader.');
+}
+
+// Include Annotation Processors for Buphalo
+require_once($PWD . '/src/AnnotationProcessors/DAO.php');
```

Comment:
consider `sprintf`
```suggestion
require_once(sprintf('%s/src/AnnotationProcessors/DAO.php', $PWD));
```

Reply:
I actually prefer the concatenation since it shouldn't ever be anything other than a precursor. I do like sprintf in a lot of cases, but here it feels less clear to me.